projects/span_finder/project.yml

```diff
@@ -0,0 +1,171 @@
+title: "Span Finder"
+description: "This project introduces a new experimental suggester that learns to predict span boundaries to produce more precise candidate spans."
+# Variables can be referenced across the project.yml using ${vars.var_name}
+vars:
+  config: "config_tok2vec" # config_tok2vec, config_trf
+  dataset: "healthsea" # healthsea, toxic, genia
+  suggester: "ngram" # ngram, span_finder
```

Comment:
Considering this project is to showcase the `span_finder` functionality, maybe that should be the default value? :-)